codec-http/src/main/java/io/netty/handler/codec/http/HttpHeadersEncoder.java

```diff
@@ -21,25 +21,29 @@
 import io.netty.util.AsciiString;
 import io.netty.util.CharsetUtil;
 
+import static io.netty.handler.codec.http.HttpConstants.*;
+import static io.netty.handler.codec.http.HttpObjectEncoder.CRLF_SHORT;
+
 final class HttpHeadersEncoder {
+    private static final int COLON_AND_SPACE_SHORT = (COLON << 8) | SP;
 
     private HttpHeadersEncoder() {
     }
 
-    static void encoderHeader(CharSequence name, CharSequence value, ByteBuf buf) throws Exception {
+    static void encoderHeader(CharSequence name, CharSequence value, ByteBuf buf) {
         final int nameLen = name.length();
         final int valueLen = value.length();
         final int entryLen = nameLen + valueLen + 4;
         buf.ensureWritable(entryLen);
         int offset = buf.writerIndex();
         writeAscii(buf, offset, name);
         offset += nameLen;
-        buf.setByte(offset ++, ':');
-        buf.setByte(offset ++, ' ');
+        buf.setShort(offset, COLON_AND_SPACE_SHORT);
```

Comment:
Do we have to account for endianness here or will the buffer implementation take care of that ( it at computer atm)